aksel-poke-dex-explorer/.editorconfig

```diff
@@ -0,0 +1,17 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.ts]
+quote_type = single
+ij_typescript_use_double_quotes = false
+
+[*.md]
+max_line_length = off
+trim_trailing_whitespace = false
```

Comment:
What is the editor you are using ? 

Unless we are all working on the same editor in a project this files are not comited to git. 


So you should remove them from git history (remote and local ) something that you need to learn to do anyway